book_source/source/module-9-miscellaneous-topics/lesson-27-reading-ethics-in-data-science/case-study-1-tracking-for-safety.md

```diff
@@ -0,0 +1,45 @@
+# Case Study 1: Tracking for Safety
+
+## Task
+
+Read both of the readings below. Reading 1 is article and Reading 2 are some tweets about the use of phone apps to track the potential spread of COVID-19. Then read the section below for [](content:module9:lesson27:case-study-1:things-to-consider). For context, this article was written relatively early during the pandemic (April 4, 2020) when quarantining was fairly ubiquitious and we didn't have a vaccine available yet to protect people at risk.
```

Comment:
Then read the section below for... - link is not showing up

Reply:
Oh interesting, the syntax they showed for linking reference sections didn't work. Fixed it!